### PR TITLE
added optional configuration parameters for the tab page selector

### PR DESCRIPTION
**Pull request by gavindoughtie**

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
@Hixie or @HansMuller are your right reviewers.

File: packages/flutter/lib/src/material/tabs.dart [hunk 1]
```diff
@@ -822,14 +822,25 @@ class _TabBarViewState extends State<TabBarView> {
...
+    this.indicatorSize: 12.0
+  }) : super(key: key);
 
   /// This widget's selection and animation state.
   ///
   /// If [TabController] is not provided, then the value of [DefaultTabController.of]
   /// will be used.
   final TabController controller;
 
+  /// The fill color for an unselected indicator. Defaults to Colors.transparent.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/tabs.dart]: `[Colors.transparent]`

Comment (Hixie) [inline packages/flutter/lib/src/material/tabs.dart]: You should add more color to these docs in a second paragraph, saying why you would set it.

**Comment by Hixie** [comment]
Tests?

**Comment by Hixie** [comment]
Gavin, and you still working on this one?

**Comment by Hixie** [comment]
I'm going to close this PR for now just to get it off our "to be reviewed" list, but please do not hesitate to reopen it once you have addressed the comments above; this patch seems eminently reasonable.

**Pull request closed without merging**